src/components/data/DataTable/DataTable.tsx

```diff
@@ -114,12 +127,73 @@ type DataTableState = {
  * DataTable component enables fetching data using predefined function and showing tabular data in a simple manner.
  *
  * ## Features
- * - data pagination
+ * - data fetch with pagination
  * - selectable rows
  * - expandable rows
  * - data sorting by columns
  * - filtering data
  *
+ * ## Data Fetch Examples
+ * This is an example of a [widget](https://github.com/cloudify-cosmo/cloudify-stage/tree/master/widgets) that uses DataTable component to easily display list of blueprints
+ * - It showcases compatibility of `fetchData` prop with `toolbox.refresh()`
+ * - `DataTable` component initial render triggers `fetchGridData` function, which fetches the data
+ * - `DataTable` component passes page size, page offset and sorting details (column name and ascending/descending information) to `fetchGridData` function. If any of these values change, the `fetchGridData` function is triggered again
+ * - `[params]` token in `fetchUrl` function is translated into grid parameters (see `fetchUrl` function section in [Widget Definition page](https://docs.cloudify.co/staging/dev/developer/writing_widgets/widget-definition/))
+ * - fetched data can be accessed in `render` method as `data` argument and presented using DataTable component
```

Comment:
After 4f4e565fcde6897538fbcdab4d1509a6a38b3a65 to be fully consistent, let's have:

```suggestion
 * This is an example of a [widget](https://github.com/cloudify-cosmo/cloudify-stage/tree/master/widgets) that uses `DataTable` component to easily display list of blueprints
 * - It showcases compatibility of `fetchData` prop with `toolbox.refresh()`
 * - `DataTable` component initial render triggers `fetchGridData` function, which fetches the data
 * - `DataTable` component passes page size, page offset and sorting details (column name and ascending/descending information) to `fetchGridData` function. If any of these values change, the `fetchGridData` function is triggered again
 * - `[params]` token in `fetchUrl` function is translated into grid parameters (see `fetchUrl` function section in [Widget Definition page](https://docs.cloudify.co/staging/dev/developer/writing_widgets/widget-definition/))
 * - fetched data can be accessed in `render` method as `data` argument and presented using `DataTable` component
```

Maybe there are more places (please check the whole file).